src/codeu/chat/client/ClientUser.java

```diff
@@ -88,27 +95,67 @@ public void showCurrent() {
     printUser(current);
   }
 
-  public void addUser(String name) {
+  //change the type to a boolean, so we can check if the user was created or not in the GUI
```

Comment:
This comment should be more about what the method does, not what you changed. Something like "Checks if the user is valid and creates a new user if so. Returns true if user was created, false otherwise." would be more descriptive.